lib/migration_generator/operation.ex

```diff
@@ -752,7 +752,7 @@ defmodule AshPostgres.MigrationGenerator.Operation do
       up(%{
         op
         | old_attribute: op.new_attribute,
-          new_attribute: op.old_attribute,
+          new_attribute: Map.put(op.old_attribute, :source, op.new_attribute.source),
```

Comment:
🤔 I think this might have undesired consequences. This is one way to do it, but another way to do it would likely be to fix the operation sorting code. There is a `defp after?` function that we should likely make return `true` to make it so that any `modify` operations come after `rename` operations.

Reply:
Ah, right you are. To solve for that in the past we've split into separate operations that only run up and down. Looking at the code I don't think there are any issues actually.